scripts/init.sql

```diff
@@ -1,158 +1,147 @@
+    url varchar(255) NOT NULL,
+    meta_data json,
+    created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    CONSTRAINT images_pkey PRIMARY KEY(id)
+);
+
+CREATE TABLE items(
+    id uuid NOT NULL DEFAULT gen_random_uuid(),
+    "name" varchar(255) NOT NULL,
+    price numeric(10, 2) NOT NULL,
+    link varchar(2000) NOT NULL,
+    retailer varchar(255) NOT NULL,
+    retailer_product_id varchar(255) NOT NULL,
+    meta_data json,
+    image_id uuid NOT NULL,
+    created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    CONSTRAINT items_pkey PRIMARY KEY(id)
+);
+
+CREATE TABLE messages(
+    id uuid NOT NULL DEFAULT gen_random_uuid(),
+    "content" varchar(500) NOT NULL,
+    sender uuid NOT NULL,
+    wishcard_id uuid NOT NULL,
+    created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    CONSTRAINT messages_pkey PRIMARY KEY(id)
+);
+
+CREATE TABLE orders(
+    id uuid NOT NULL DEFAULT gen_random_uuid(),
+    status integer NOT NULL DEFAULT 0,
+    delivery_date timestamp,
+    tracking_info varchar(500),
+    donor_id uuid NOT NULL,
+    child_id uuid NOT NULL,
+    item_id uuid NOT NULL,
+    agency_id uuid NOT NULL,
+    created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at timestamp,
+    CONSTRAINT orders_pkey PRIMARY KEY(id)
+);
+
+CREATE TABLE users(
+    id uuid NOT NULL DEFAULT gen_random_uuid(),
+    first_name varchar(255) NOT NULL,
+    last_name varchar(255) NOT NULL,
+    email varchar(255) NOT NULL,
+    "password" varchar(100) NOT NULL,
+    "role" integer NOT NULL,
+    login_mode integer NOT NULL,
+    bio varchar(500),
+    verified boolean NOT NULL DEFAULT false,
+    image_id uuid NOT NULL,
+    created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at timestamp,
+    deleted_at timestamp,
+    CONSTRAINT users_pkey PRIMARY KEY(id)
+);
+
+CREATE TABLE verification_tokens(
+    id uuid NOT NULL DEFAULT gen_random_uuid(),
+    token varchar(255) NOT NULL,
+    "type" integer NOT NULL,
+    expiration timestamp NOT NULL,
+    user_id uuid NOT NULL,
+    created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    CONSTRAINT verification_tokens_pkey PRIMARY KEY(id)
+);
+
+CREATE TABLE verifications(
+    id uuid NOT NULL DEFAULT gen_random_uuid(),
+    email_verified boolean NOT NULL,
+    user_id uuid NOT NULL,
+    created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at timestamp,
+    CONSTRAINT verifications_pkey PRIMARY KEY(id)
+);
+
+CREATE TABLE wishcards(
+    id uuid NOT NULL DEFAULT gen_random_uuid(),
+    address_line_1 varchar(255) NOT NULL,
+    address_line_2 varchar(255),
+    city varchar(100) NOT NULL,
+    state varchar(100) NOT NULL,
+    country varchar(100) NOT NULL,
+    zip_code varchar(50) NOT NULL,
+    status integer NOT NULL DEFAULT 0,
+    created_by uuid NOT NULL,
+    agency_id uuid NOT NULL,
+    child_id uuid NOT NULL,
+    item_id uuid NOT NULL,
+    image_id uuid NOT NULL,
+    order_id uuid NOT NULL,
```

Comment:
```suggestion
    agency_id uuid NOT NULL REFERENCES agencies (id),
    child_id uuid NOT NULL REFERENCES children (id),
    item_id uuid NOT NULL REFERENCES items (id),
    image_id uuid NOT NULL REFERENCES images (id),
    order_id uuid NOT NULL REFERENCES orders (id),
```

added agency, child, item, image and order foreign key constraints